frontend/src/pages/notebookController/NotebookController.tsx

```diff
@@ -68,7 +74,20 @@ export const NotebookController: React.FC = React.memo(() => {
       }
     };
     checkUserState().catch((e) => console.error(e));
-  }, [username, dashboardConfig, namespace]);
+  }, [username, dashboardConfig]);
+
+  const notebookNamespace = React.useMemo(
+    () => dashboardConfig.spec.notebookController?.notebookNamespace,
+    [dashboardConfig],
+  );
+
+  React.useEffect(() => {
+    if (notebookNamespace && namespace) {
+      validateNotebookNamespaceRoleBinding(notebookNamespace, namespace).catch((e) =>
+        console.error(e),
```

Comment:
Do we not want to showcase the error? If we don't validate, wouldn't this be a problem for the user?

Reply:
Updated.